server/views/formPages/recat/securityInput.html

```diff
@@ -40,12 +44,12 @@ <h1 id="securityInputHeader" class="govuk-heading-l">{{ pageTitle }}</h1>
     classes: "govuk-radios--inline",
     fieldset: {
       legend: {
-        text: "Do you want to request additional information from the security team?",
+        text: "Do you want to include a note to security?",
```

Comment:
Not sure about the data saved off when you submit the page. I thought the value of 'securityInputNeeded' should remain 'yes' (realised maybe as a hidden field on the form) and "Do you want to include a note to security" should be a new field introduced as part of this ticket. 'securityInputNeededText' might be all right how it is, as that is a generic note field. Happy to discuss if needed.